### ISSUE TITLE
autoupdate brim to point to zq masters

### ISSUE THREAD

**Issue by mikesbrown** [brimsec/brim#734]
This can happen once part one of #634 is finished.

**Comment by mikesbrown** [comment]
@alfred-landrum requested we not have the zqbot open PRs for every zq change.

Here's what I think the logic should be:
1. When a PR is merged to zq master, dispatch that information to Brim
2. If Brim's package.json points to zq at a version tag, this means a release is immiment. Instead of auto-committing, open a PR to allow manual approval of the zq version bump pending CI success.
3. If Brim's package.json does not point to zq at a version tag, run brim tests, and if they succeed, autocommit/push package.json and package-lock.json. Handling failure here is for another issue, I think.

**Comment by philrz** [comment]
Should I just augment the release process such that I get PR approved/merged to get the `zq` pointer in Brim's `package.json` back into "hash mode" right after a release goes out successfully? If we leave the step 2 to the random lucky winner who first commits to Brim after a release, I could imagine people forgetting to deal with an auto-generated PR and slowing things down, which is why I'd be inclined to just take care of it proactively.

**Comment by mikesbrown** [comment]
If you'd rather.  I don't have a very strong feeling either way. This was a strawman. We can always adjust the process.
